app/services/content_item_publisher.rb

```diff
@@ -1,10 +1,48 @@
 class ContentItemPublisher
   def publish(flow_presenters)
+    find_smart_answers_without_start_pages(flow_presenters)
     flow_presenters.each do |smart_answer|
-      content_item = FlowContentItem.new(smart_answer)
-      Services.publishing_api.put_content(content_item.content_id, content_item.payload)
-      Services.publishing_api.publish(content_item.content_id, 'minor')
+      content_item = assign_content_item(smart_answer)
+      if content_item
+        Services.publishing_api.put_content(content_item.content_id, content_item.payload)
+        Services.publishing_api.publish(content_item.content_id, 'minor')
+      end
     end
+    republish_start_page
+  end
+
+  def find_smart_answers_without_start_pages(flow_presenters)
```

Comment:
By defining a predicate an extract attr accessor field and may be a predicate method in the `lib/smart_answer/flow.rb` that will identify a smart answer with a start page 

So you flow will look something like:

```ruby
module SmartAnswer
  class PartYearProfitTaxCreditsStartPageFlow < Flow
    def define
      start_page = true
      name 'part-year-profit-tax-credits'

      status :published
      satisfies_need "103438"
      content_id "de6723a5-7256-4bfd-aad3-82b04b06b73e"

```

````ruby

module SmartAnswer
  class Flow
    attr_reader :nodes
    attr_accessor :status, :need_id :start_page

    def start_page?
       start_page
    end 
```